### PR TITLE
chore(workspace): upgrade nx to 16.5.5 (VIV-1181)

### PR DESCRIPTION
**Pull request by YonatanKra**

### CODE REVIEW

File: libs/components/src/lib/accordion/ui.test.ts [hunk 1]
```diff
@@ -18,8 +18,8 @@ test('should show the component', async ({ page }: { page: Page }) => {
 		''
 	);
 
-	page.setViewportSize({ width: 500, height: 720 });
-
+	await page.setViewportSize({ width: 500, height: 720 });
+	await page.pause();
```
Comment (rachelbt) [inline libs/components/src/lib/accordion/ui.test.ts:22]:
```suggestion

```

Comment (rachelbt) [reply libs/components/src/lib/accordion/ui.test.ts:22]: why is this needed? you didn't add this in the other ui tests

File: libs/components/src/lib/accordion/ui.test.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ test('should show the component', async ({ page }: { page: Page }) => {
 	const testWrapper = await page.$('#wrapper');
 
 	await page.waitForLoadState('networkidle');
-
+	await page.evaluate(() => document.fonts.ready);
```
Comment (rachelbt) [inline libs/components/src/lib/accordion/ui.test.ts:35]: why only this has it and not all other components?

File: libs/components/src/lib/action-group/action-group.ts [hunk 3]
```diff
@@ -60,7 +60,7 @@ export class ActionGroup extends FoundationElement {
 		mode: 'boolean',
 	}) tight = false;
 
-	@attr() role: string | undefined;
+	@attr() override role: string | null = null;
 	@attr({attribute: 'aria-label'}) override ariaLabel: string | null = null;
 }
```
Comment (rachelbt) [inline libs/components/src/lib/action-group/action-group.ts:65]: action-group is missing the safari png

**Review by rachelbt — COMMENTED** [review]
Left you 2 comments.
went over all the snap shote - all seems OK. Spezia is smaller and more condensed. 
I looked at the other files - there are a lot! guess its upgrading all those files.
I think it is OK.
The docs look OK

**Review by rachelbt — COMMENTED** [review]
Went over all the PNG's they look OK. The font had changed to Spezia and render slightly different in the terms of stretch and line height.
UI - wise it looks OK.

**Review by RichardHelm — APPROVED** [review]

**Pull request merged**